lib/core/interfaces/IOperationRateLimiter.ts

```diff
@@ -0,0 +1,19 @@
+import TransactionModel from '../../common/models/TransactionModel';
+
+/**
+ * Rate limits operations. only allow n number of highest fee transactions to pass.
+ */
+export default interface IOperationRateLimiter {
```

Comment:
Discussion: I wonder if we should generalize this name to something like `IThroughputLimiter` or we only foresee rate limiting by operation forever.

Reply:
Good idea. It doesn't have to be operation.